.github/workflows/SQL2022.yml

```diff
@@ -131,6 +154,12 @@ jobs:
           @input_data_1 =N'SELECT 1 AS hello'
           WITH RESULT SETS (([hello] int not null));"
 
+      - name: Execute SPEES for Python
+        run: sqlcmd -S localhost -U SA -P %dbPassword% -l 5 -Q "
+          EXEC sp_execute_external_script  @language =N'Python',
+          @script=N'OutputDataSet = InputDataSet;',
+          @input_data_1 =N'SELECT 1 AS hello' WITH RESULT SETS (([hello] int not null));"
```

Comment:
do we fail the pipeline if this step fails?

Reply:
It will fail if there is an error.